internal/karma/karma.conf.js

```diff
@@ -1,5 +1,59 @@
 // Karma configuration
 // GENERATED BY Bazel
+const path = require('path');
+const fs = require('fs');
+const tmp = require('tmp');
+
+process.env.CHROME_BIN = require('puppeteer').executablePath();
+
+let files = [
+  TMPL_files
+];
+
+const manifestFile = path.join(process.env.TEST_SRCDIR, "MANIFEST");
+if (fs.existsSync(manifestFile)) {
+  const manifest = {};
+  fs.readFileSync(manifestFile, { encoding: 'utf8' })
+    .split('\n')
+    .forEach((l) => {
+      const m = l.split(' ');
+      manifest[m[0]] = m[1];
+    });
+  const manifestKeys = Object.keys(manifest);
+  files = files.map((f) => {
+    if (manifestKeys.includes(f)) {
+      return manifest[f];
+    } else {
+      throw new Error(`File not found in MANIFEST: ${f}`);
+    }
+  });
+} else {
+  files = files.map((f) => f.replace('build_bazel_rules_typescript/external/', ''));
+}
+
+let requireFiles = [
+  TMPL_files
+].map((f) => f.replace('build_bazel_rules_typescript/external/', ''));
```

Comment:
why is rules_typescript special here? Could there be other external repos in the files?

Reply:
that is to handle the two files from ts_web_test.bzl: ```  files_entries += [
    "build_bazel_rules_typescript/external/build_bazel_rules_typescript_karma_deps/node_modules/requirejs/require.js",
    "build_bazel_rules_typescript/external/build_bazel_rules_typescript_karma_deps/node_modules/karma-requirejs/lib/adapter.js",
  ]
```
but yes, I suppose there could be other externals included so should change the replace to handle any external workspace

Reply:
Oh. I took a closer look at the MANIFEST file. I think this code can be improved to get rid the `build_bazel_rules_typescript/external/` bit. Will try it out.

Reply:
Cleaned this up. No `build_bazel_rules_typescript/external/` prefix is necessary.